test/bdd/pkg/vault/vault_steps.go

```diff
@@ -249,6 +249,9 @@ func (e *Steps) createVault(endpoint string) error {
 
 	e.bddContext.VaultID = result.ID
 
+	// to prevent an error "failed to resolve did: DID does not exist"
+	time.Sleep(time.Second)
```

Comment:
you need to add resolve DID https://github.com/trustbloc/edge-service/blob/main/test/bdd/pkg/vc/vc_steps.go#L232 instead of sleep